components/script/dom/writablestreamdefaultcontroller.rs

```diff
@@ -389,6 +398,17 @@ impl WritableStreamDefaultController {
         self.underlying_sink_obj.set(*this_object);
     }
 
+    pub(crate) fn signal_abort(
```

Comment:
We could document this as being the "Signal abort" call from https://streams.spec.whatwg.org/#writable-stream-abort